File: inv_java/src/main/java/inv_java/weixin/Message.java
package inv_java.weixin;

public class Message {
	
	public String msg;
	
	public Message(String msg) {
		this.msg = msg;
	}
	
	public void send() {
        Weixin.send(this);
	}
	
	@Override
	public String toString() {
		return msg;
	}

}

File: inv_node/models/Message.js
/**
 * Created by wupei on 16/9/25.
 */

class Message {

    constructor(runner) {
        this.message = runner.message;
        this.touser = runner.touser;
    }

    send() {
        var weixin = require('../core/weixin')
        weixin.send(this);
    }

}

module.exports = Message;